src/Nowruz/modules/settings/components/orgTeam/useOrgTeam.tsx

```diff
@@ -88,35 +88,34 @@ export const useOrgTeam = () => {
     try {
       if (searchText) {
         const response = await filterFollowings({ name: searchText, type: 'users', page: 1, limit: 10 });
-        const filtered = response.items.filter((item) => !teamMembers.map((m) => m.id).includes(item.identity_meta.id));
+        const filtered = response.items.filter(item => !teamMembers.map(m => m.id).includes(item.identity_meta.id));
         cb(followingToOption(filtered));
       }
     } catch (error) {
       console.error('Error fetching following data:', error);
     }
   };
 
-  const onSelectMember = (member) => {
-    setselectedPerson(member);
-    const members = addedMembers.filter((item) => !item);
-    members.push(member.value);
+  const onSelectMember = member => {
+    const members = addedMembers.filter(item => !item);
+    members.push(member);
     setAddedMembers(members);
   };
 
   const handleAddAnother = () => {
     const members = [...addedMembers];
-    members.push({ value: '', label: '' });
+    members.push({ value: undefined, label: undefined });
```

Comment:
`For me`: what is this line? i can't get why we push empty value in members ?

Reply:
![image](https://github.com/socious-io/web-app-v2/assets/60564928/c3623d31-14cf-4ed9-9ce2-6ffcfb1005a4)
In fact by clicking on Add another, an empty value is pushed to display an empty SearchDropDown which can be filled later